docs/automated-testing/ui-testing/teams-tests.md

```diff
@@ -0,0 +1,240 @@
+
+    Solution to fix it: 
+    1. [Enable Windows Hypervisor Platform](https://devblogs.microsoft.com/visualstudio/hyper-v-android-emulator-support/)
+    1. [Enable Hyper-V](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v#enable-the-hyper-v-role-through-settings) (if not enabled by default)
+1. Restart the VM.
+1. Restart the AVD.
+
+
+### How to inspect the Teams app in Azure Virtual Device (AVD)
+Inspecting the app is very useful as you write new tests, as it allows you to find the ID of different elements displayed on the screen - similar to DevTools, which allows you to navigate through the DOM of a web page.
+
+[Appium Inspector](https://inspector.appiumpro.com/) is a very useful tool that allows you to inspect an app runing on an emulator.
+
+> Note: This section assumes that you have already performed the prerequisites from **How to run mobile end to end tests on a Windows machine**
+
+### Steps to inspect Teams app:
+1. Run the appium server with [--alow-cors flag](https://appium.readthedocs.io/en/latest/en/writing-running-appium/server-args/) by running the following command in a terminal:
+    ```
+    appium --allow-cors
+    ```
+2. Go to https://inspector.appiumpro.com and type in the following properties:
+    ```json
+    {
+    "appium:deviceName": "your-emulator-name",
+    "appium:appPackage": "com.microsoft.teams",
+    "appium:appActivity": "com.microsoft.skype.teams.Launcher",
+    "appium:automationName": "UiAutomator2",
+    "platformName": "Android"
+    }
+    ```
+
+    1. "appium:deviceName" - is the name of your emulator. In **Useful commands** sections from above, you can see how to get the name of your AVD.
+    1. "appium:appPackage" - is the name of the package, should be kept to "**com.microsoft.teams**".
+    1. "appium:appActivity"-  is the name of the activity in the app that you want to launch, should be kept to "**com.microsoft.skype.teams.Launcher**"
+    1. "appium:automationName" - is the name of the driver you are using, in this case, "**UiAutomator2**"
+
+    If the appium server runs on your local machine at the default portal, then Remote Host and Remote Port can be kept to the default values.
+
+    The configuration should look similar to the printscren below:
+    ![appium-inspector](./images/appium-inspector.png)
+3. Press on **Start Session**.
+    - In the browser, you should see a similar view as below:
+    ![teams-appium-inspector](./images/teams-appium-inspector.png)
+    - You can do any action on the emulator, and if you press on the "Refresh" button in the browser, the left hand side of the Appium Inspector will reflect your app. In the **App Source** you will be able to see the IDs of the elements, so you can write relevant selectors in your tests. 
+
+#### How to connect to Appium server
+We will provide an example on how to achieve this using JavaScript. Similar approach can be followed for other languages.
```

Comment:
```suggestion
#### Connecting to Appium server

Below is how this can be accomplished with JavaScript. A similar approach can be followed for other languages.
```